tests/parametric/test_otel_span_methods.py

```diff
@@ -110,13 +107,21 @@ def test_otel_set_attributes_different_types(self, test_agent, test_library):
             assert root_span["meta"]["d_bool_val"] == "false"
             assert root_span["meta"]["array_val_int"] == "[10, 20]"
             assert root_span["meta"]["array_val_double"] == "[10.1, 20.2]"
-        elif root_span["meta"]["language"] == "dotnet":
+        elif root_span["meta"]["language"] in "dotnet":
```

Comment:
Typo ?

Reply:
This was `root_span["meta"]["language"] in ["dotnet", "ruby"]` earlier in this PRs lifetime, I'll revert this line.